qiskit_experiments/calibration_management/backend_calibrations.py

```diff
@@ -43,8 +44,33 @@ class BackendCalibrations(Calibrations):
     __qubit_freq_parameter__ = "qubit_lo_freq"
     __readout_freq_parameter__ = "meas_lo_freq"
 
-    def __init__(self, backend: Backend):
-        """Setup an instance to manage the calibrations of a backend."""
+    def __init__(
+        self,
+        backend: Backend,
+        basis_gates: Optional[List[str]] = None,
+        library: BasisGateLibrary = None,
```

Comment:
Do you think it is best for the `BasisGateLibrary` to always contain all of the gates coded into it? One modification that could be made here is to make `basis_gates` an argument to the `BasisGateLibrary`'s `__init__` method and have it do the gate filtering on creation. Then `BackendCalibrations` would only need to take the `library` here.

Reply:
+1

Reply:
Good point. See here: d3c0b69